tests/test_choose_disk.py

```diff
@@ -0,0 +1,159 @@
+import imp
+import inspect
+import mock
+import os
+from nose.tools import assert_equals, nottest
```

Comment:
+1 ``py.test`` would be much preferred here, ``py.test`` is also what we're using in the CI tests.